frontend-ssr/components/lessons/LessonComments/comment/CommentBlock.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { timeAgo } from '../../../../services/Utils';
+
+import styles from './CommentBlock.module.scss';
+
+interface Props {
+  avatar: string;
+  username: string;
+  name: string;
+  comment: string;
+  timestamp: number;
+}
+
+const CommentBlock = ({
+  avatar,
+  username,
+  name,
+  comment,
+  timestamp,
+}: Props) => (
+  <li className={styles.comment}>
+    <div>
+      <div className="d-flex">
+        <a href="#" className="d-flex no-underline">
+          <img
+            width="32"
+            height="32"
+            src={avatar}
+            alt={`${username} avatar`}
+          />
+          <span className={styles.user}>{name || username}</span>
+        </a>
+        <span> a lasat urmatorul comentariu: </span>
```

Comment:
I looked over this UI and let's

1. Let's make the image a little bigger
2. The image should be in a link that leads to the user profile
3. Instead of the "a lasat urmatorul comentariu" let's put the username - also a link to the user profile. Here maybe you can combine them in the same link (the image and the username text)